mlir/include/mlir/IR/BuiltinTypes.h

```diff
@@ -39,6 +39,16 @@ struct IntegerTypeStorage;
 struct TupleTypeStorage;
 } // namespace detail
 
+/// Type trait indicating that the type can be an operand to an elementwise op.
+template <typename ConcreteType>
+class MappableContainer
+    : public TypeTrait::TraitBase<ConcreteType, MappableContainer> {};
```

Comment:
Do we need this right now knowing we have the ShapedTypeInterface?

Reply:
We don't. Good catch, Mehdi!